packages/flutter_tools/test/general.shard/build_system/source_test.dart

```diff
@@ -233,9 +232,5 @@ void main() {
     fizzSource.accept(visitor);
 
     expect(visitor.sources.single.path, contains('engine.version'));
-    verifyNever(artifacts.getArtifactPath(
```

Comment:
This is already tested by the fact that the above call does a `.single` which would fail if there were another dependency or if the artifact path was used instead of the engine version